e2e/e2e.go

```diff
@@ -299,16 +302,37 @@ func main() {
 		log.Fatal().Err(err).Msg("Failed to run devconfig")
 	}
 
-	log.Info().Msg("Starting devenv tunnel")
-	cmd = exec.CommandContext(ctx, "devenv", "--skip-update", "tunnel")
-	cmd.Stderr = os.Stderr
-	cmd.Stdout = os.Stdout
-	cmd.Stdin = os.Stdin
-	err = cmd.Start()
-	if err != nil {
-		log.Fatal().Err(err).Msg("Failed to start devenv tunnel")
+	var killLocalizer bool
+	if !localizer.IsRunning() {
+		killLocalizer = true
+
```

Comment:
My last comment here seems to have been dropped on the floor by github :(

Anyway, I think we should use the standard defer/cleanup pattern here (maybe using [cleanup](https://engdocs.outreach.cloud/github.com/getoutreach/gobox/pkg/cleanup):

```suggestion
	var killLocalizer func()
	cleanups := cleanup.Funcs{&killLocalizer} // can add more cleanups later on if we need to
	defer cleanups.Run()
	
	if !localizer.IsRunning() {
		killLocalizer = func() { killProcByName("localizer") }
		... start localizer... 
	}
```

Reply:
Yeah I'll do this, sounds reasonable

Reply:
there ended up being not a reason to do this quite yet, really not much to cleanup other than localizer